{
    "config": {
        "abort": {
            "single_instance_allowed": "Ji\u017e nastaveno. Je mo\u017en\u00e9 jen jedin\u00e9 nastaven\u00ed."
        },
        "error": {
            "cannot_connect": "Nepoda\u0159ilo se p\u0159ipojit",
            "unknown": "Neo\u010dek\u00e1van\u00e1 chyba"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Hostitel",
                    "password": "Heslo",
                    "username": "U\u017eivatelsk\u00e9 jm\u00e9no"
                },
                "data_description": {
                    "host": "N\u00e1zev hostitele nebo IP adresa hlavn\u00edho opakova\u010de Lutron."
                },
                "description": "Zadejte p\u0159ihla\u0161ovac\u00ed \u00fadaje hlavn\u00edho opakova\u010de",
                "title": "Nastaven\u00ed hlavn\u00edho opakova\u010de"
            }
        }
    },
    "issues": {
        "deprecated_yaml_import_issue_cannot_connect": {
            "description": "Nastaven\u00ed Lutronu pomoc\u00ed YAML se odeb\u00edr\u00e1, ale p\u0159i importu va\u0161eho nastaven\u00ed YAML do\u0161lo k chyb\u011b p\u0159ipojen\u00ed.\n\nV\u011bci, kter\u00e9 m\u016f\u017eete vyzkou\u0161et:\nUjist\u011bte se, \u017ee v\u00e1\u0161 dom\u00e1c\u00ed asistent dos\u00e1hne na hlavn\u00ed opakova\u010d.\nRestartujte hlavn\u00ed opakova\u010d odpojen\u00edm na 60 sekund.\nZkuste se p\u0159ihl\u00e1sit do hlavn\u00edho opakova\u010de na IP adrese, kterou jste zadali ve webov\u00e9m prohl\u00ed\u017ee\u010di, a se stejn\u00fdmi p\u0159ihla\u0161ovac\u00edmi \u00fadaji.\n\nPot\u00e9 restartujte Home Assistanta a zkuste importovat tuto integraci znovu.\n\nP\u0159\u00edpadn\u011b m\u016f\u017eete nastaven\u00ed Lutronu z va\u0161eho nastaven\u00ed YAML \u00fapln\u011b odebrat, restartovat Home Assistanta a p\u0159idat integraci Lutron ru\u010dn\u011b.",
            "title": "Import nastaven\u00ed Lutron YAML se nem\u016f\u017ee p\u0159ipojit k serveru"
        },
        "deprecated_yaml_import_issue_unknown": {
            "description": "Nastaven\u00ed Lutronu pomoc\u00ed YAML se odeb\u00edr\u00e1, ale p\u0159i importu va\u0161eho st\u00e1vaj\u00edc\u00edho nastaven\u00ed do\u0161lo k nezn\u00e1m\u00e9 chyb\u011b.\nInstalace nebude pokra\u010dovat.\n\nKonkr\u00e9tn\u00ed chybu najdete v protokolech. Nejpravd\u011bpodobn\u011bj\u0161\u00ed p\u0159\u00ed\u010dinou je chyba s\u00edt\u011b nebo hlavn\u00ed opakova\u010d nefunguje nebo m\u00e1 neplatn\u00e9 nastaven\u00ed.\n\nOv\u011b\u0159te, \u017ee v\u00e1\u0161 syst\u00e9m Lutron funguje spr\u00e1vn\u011b a restartujte Home Assistanta, abyste se pokusili o import znovu.\n\nP\u0159\u00edpadn\u011b m\u016f\u017eete nastaven\u00ed Lutronu z va\u0161eho nastaven\u00ed YAML \u00fapln\u011b odebrat, restartovat Home Assistanta a p\u0159idat integraci Lutron ru\u010dn\u011b.",
            "title": "Po\u017eadavek na import nastaven\u00ed Lutron YAML se nezda\u0159il kv\u016fli nezn\u00e1m\u00e9 chyb\u011b"
        }
    }
}